vendor/brightray/browser/network_delegate.cc

```diff
@@ -144,4 +144,24 @@ bool NetworkDelegate::OnCancelURLRequestWithPolicyViolatingReferrerHeader(
   return false;
 }
 
+bool NetworkDelegate::OnCanQueueReportingReport(
+    const url::Origin& origin) const {
+  return true;
```

Comment:
I think I'm just going to dump this class and replace with network_delegate_impl